### PR TITLE
Move from Gitlens to the library the columns settings button

### PR DESCRIPTION
**Pull request by ericf-axosoft**
## Summary of changes:
- Removed graph header settings button from Gitlens as now the Graph component can display that button and we have exposed new input properties.

## Notes
- This PR works with GK graph library side changes: https://github.com/gitkraken/GitKrakenComponents/pull/231
- Before merge this PR, we have to update the GK graph library version dependency.

## Tasks
[230](https://github.com/gitkraken/GitKrakenComponents/issues/230)
Internal task: `GK.3665`

### CODE REVIEW

**Review by axosoft-ramint — COMMENTED** [review]
Looks good to me, but since we are releasing GitLens tomorrow, let's hold off on merging this in. Since this proves it can work with the settings button in the graph component, I think you can merge the component-side PR and update the library version so you can use it in GKC @ericf-axosoft

**Review by Haitham103 — APPROVED** [review]

**Review by axosoft-ramint — APPROVED** [review]
Include dependency bump before merge.

File: src/webviews/apps/plus/graph/GraphWrapper.tsx
Comment (eamodio) [inline src/webviews/apps/plus/graph/GraphWrapper.tsx:1427]: This was removed, but how can GitLens provide the `data-vscode-context` now? @ericf-axosoft @axosoft-ramint

Comment (ericf-axosoft) [reply src/webviews/apps/plus/graph/GraphWrapper.tsx:1427]: The graph sets the `data-vscode-context` for the button by using the actual `contexts.settings`.

Comment (axosoft-ramint) [reply src/webviews/apps/plus/graph/GraphWrapper.tsx:1427]: Yep, we pass `context` to the graph as a prop `contexts` and `context.settings` makes it to the button via the prop as `contexts.settings`

**Pull request merged**
